### PR TITLE
Integrate SharedModels

### PR DESCRIPTION
**Pull request by terlan98**
This PR involves the replacement of models from artemis-ios-core into Themis.

### CODE REVIEW

File: Themis.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -110,11 +123,22 @@
...
+		653452B52A068E4D008D9598 /* ParticipationExtension.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ParticipationExtension.swift; sourceTree = "<group>"; };
+		6564DC122A00F02900CF2F89 /* ExerciseStatistics.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ExerciseStatistics.swift; sourceTree = "<group>"; };
+		6564DC162A00F15400CF2F89 /* ExerciseStatsForAssessmentDashboard.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ExerciseStatsForAssessmentDashboard.swift; sourceTree = "<group>"; };
 		6567A90229EFF85200A5EFDA /* ScorePicker.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ScorePicker.swift; sourceTree = "<group>"; };
+		656EAF5F2A0256A6003CF2B4 /* AssessmentFeedback.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AssessmentFeedback.swift; sourceTree = "<group>"; };
+		658057142A02C1CE00835AD0 /* artemis-ios-core-modules */ = {isa = PBXFileReference; lastKnownFileType = wrapper; name = "artemis-ios-core-modules"; path = "../artemis-ios-core-modules"; sourceTree = "<group>"; };
```
Comment (maximiliansoelch) [inline Themis.xcodeproj/project.pbxproj]: Is this needed?
Screenshot 2023-05-11 at 17 44 52

Comment (terlan98) [reply Themis.xcodeproj/project.pbxproj]: Nope, I'll remove it now

Comment (terlan98) [reply Themis.xcodeproj/project.pbxproj]: Removed. Thanks for noticing 🙏

**Review by maximiliansoelch — APPROVED** [review]
Tested locally, app still works as expected. 
Changes look good 👍

**Pull request merged**